cypress/support/pages/dashboard/pageCommondashboard.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+class Dashboard_Common {
```

Comment:
Dashboard_Common -> Common_Dashboard
Similar to comment in cypress/support/pages/courses/pageCommonCourse.js.